pom.xml

```diff
@@ -1,8 +1,8 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
   <modelVersion>4.0.0</modelVersion>
   <groupId>com.axonivy.market</groupId>
```

Comment:
The groupId is quite generic. We recommend to use com.axonivy.market.utils.persistence.

Reply:
I'd skip the 'market' part: e.g. `com.axonivy.utils.persistence-utils`

Reply:
'-' is not a valid character in the group id

Reply:
I would go for ```com.axonivy.utils.persistence```